apps/engagement-app/src/components/AppDetails.tsx

```diff
@@ -1,44 +1,54 @@
-import React, { useEffect, useState } from "react"
-import { useParams } from "react-router-dom"
-import { useAccount } from "wagmi"
-import { useEngagementRewards, type RewardEvent } from "@GoodSDKs/engagement-sdk"
-import { Card, CardContent, CardHeader, CardTitle } from "./ui/card"
-import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "./ui/table"
-import { formatEther } from "viem"
-import { Loader2 } from "lucide-react"
-import env from "@/env"
-import { TruncatedAddress } from "./ui/TruncatedAddress"
+import React, { useEffect, useState } from "react";
```

Comment:
when files outside your scope of work changes, it is a cause for alert
specifically it makes it very hard to review
dont commit global style changes
change your lint to not use ';' at the end

Reply:
Now globally set in prettierrc to ;